docs/zh-CN/faq/error.md

```diff
@@ -0,0 +1,147 @@
+---
```

Comment:
You need to add this new docs to the `docs/.vuepress/sidebar.js`
And also need to update the english doc.